File: Ng/src/app/task.ts
export class Task {
    id: string;
    title: string;
    description: string;
    isComplete: boolean;
    dueDate: Date;
    parentId: string;    

    constructor(title: string, id?: string, description?: string) {
        this.id = id;
        this.title = title;
        this.description = description;
    }
}

File: Python/task.py
from datetime import date

class Task(object):

    _id = ""
    _title = ""
    _description = ""
    _dueDate = date.today()
    _isComplete = False

    @property
    def id(self):
        return self._id 
    
    @id.setter
    def id(self, id ):
        self._id = id

    @property
    def title(self):
        return self._title 
    
    @title.setter
    def title(self, title):
        self._title = title

    